Notifier.py

```diff
@@ -0,0 +1,25 @@
+import os
+import smtplib
+
+
+
+
+class Notifier:
+    # Hardcoded email address to test notifier
+    subscribers = {
+        'Johannes': 'schmid.johannes90@gmail.com',
+        'hannesxo': 'hannes322@yahoo.de'
+
+    }
+
+    def notify(self, msg):
+        for subscriber in self.subscribers:
+            sender_email = 'festify.alerts@gmail.com'
+            sender_password = os.environ.get('senderpassword')
+            receiver_mail = self.subscribers[subscriber]
+            server = smtplib.SMTP('smtp.gmail.com', 587)
```

Comment:
Since this part is always the same it's not nice that it's inside the `notify()` method, because every time you want to send a message you're creating the server and doing the login from scratch.
A cleaner way would be to move this logic to the constructor. In this way the server is only created and authenticated when you create the Notifier instance, and then you can just send new emails.